.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r .reqs/test.txt
+          pip install -r dev/${{ matrix.python-version }}/requirements-test.txt
```

Comment:
For me the dirname `dev` is a bit misleading. `env(s)` or `req(s)` sounds better to me. What was the intention behind `dev`?

Reply:
Yes that's true. Initially I thought to put all things related to `dev` in there. Ideally that would also mean all these config files like `.pylintrc`, but that's not how these tools work (unless you specify every config file with a flag). For the same reason, I also initially named the workflow `ci-dev.yml`.
I'll go for `req`, I think. Some tools use `requirements` (e.g. [`pip-tools-multi`](https://pypi.org/project/pip-compile-multi/)), but then the full paths become rather repetitive.

Reply:
1c75aa2